src/rest/Plan.ts

```diff
@@ -30,3 +30,23 @@ export interface IPlan {
     largeExpression: string;
   };
 }
+
+/**
+ * Describes the plan of execution of a search
+ */
+export class ExecutionPlan {
+  constructor(private response: IPlanResponse) {}
+
+  public get basicExpression() {
+    return this.response.parsedInput.basicExpression;
+  }
+
+  public get largeExpression() {
+    return this.response.parsedInput.largeExpression;
+  }
+
+  public get redirectionURL() {
```

Comment:
Suggestion:

Gets the URL to redirect the browser to, if the search request satisfies the condition of a `redirect` trigger rule in the query pipeline.

Returns `null` otherwise.